tsconfig.json

```diff
@@ -5,7 +5,7 @@
     "esModuleInterop": true,
     "experimentalDecorators": true,
     "jsx": "react",
-    "lib": ["es2016", "es2019", "es2020.promise", "dom", "dom.iterable", "ES2021.String"],
+    "lib": ["es2016", "es2019", "es2020.promise", "dom", "dom.iterable", "es2021.string"],
```

Comment:
do we actually need the lib? and if yes, why not simply es2021 and that it?

Reply:
@metalice I will check if we still need these libs.